src/django/pfb_analysis/serializers.py

```diff
@@ -1,38 +1,58 @@
+from collections import OrderedDict
+
 from rest_framework import serializers
 
 from pfb_analysis.models import AnalysisJob, Neighborhood
 from pfb_network_connectivity.serializers import PFBModelSerializer
 from rest_framework_gis.serializers import GeoFeatureModelSerializer
 
 
-class AnalysisJobSerializer(PFBModelSerializer):
+class PrimaryKeyReferenceRelatedField(serializers.PrimaryKeyRelatedField):
+    """ A custom relational field for read-only objects
```

Comment:
I feel like docstring style diversity in the wild is somewhat high, and I had gotten muddled from seeing various patterns that differ slightly but are all obviously intentional.  I've now perused PEP 257 again and attempted to bring this file (not the project, just the file I happen to be modifying) into line with it.